### PR TITLE
fetched foundationJobs in join page

### PR DESCRIPTION
**Pull request by ertugrul-aydin**

### CODE REVIEW

**Comment by ertugrul-aydin** [comment]
How can we fetch jobs related to the necessary filtering feature when pulling jobs from Strapi?

File: packages/ui/src/templates/JoinTemplate/JoinTemplate.tsx [hunk 1]
```diff
@@ -44,6 +44,14 @@ export const JoinTemplate: FC<JoinTemplateProps> = ({ title }) => {
 
   const platforms = platformsResult.data?.data || []
 
+  // Fetch foundation jobs
+  const foundationJobsResult = useStrapiRequest<Job>({
+    endpoint: 'jobs',
+    locale,
```
Comment (7alip) [inline packages/ui/src/templates/JoinTemplate/JoinTemplate.tsx:50]: @ertugrul-aydin Did you try filtering jobs by platform here?

- `filters: { platform: { $null: true } }` or
- `filters: { platform: { $eq: null } }`

Comment (ertugrul-aydin) [reply packages/ui/src/templates/JoinTemplate/JoinTemplate.tsx:50]: I noticed there were some jobs in Strapi without any platform linked to them, so the filter wasn't working. Now, after adding those jobs without any platform linked, they're showing up under Foundation.

File: packages/ui/src/components/JoinForm/JoinForm.tsx [hunk 2]
```diff
@@ -198,6 +199,23 @@ export const JoinForm: FC<JoinFormProps> = ({
...
+                key={job.id}
+                id={job.id.toString()}
+                {...register(`jobs`)}
+                value={job.id}
+                textTransform={'capitalize'}
+              >
+                {job[`name_${locale}`]}
+              </Checkbox>
+            </Stack>
+          ))}
```
Comment (7alip) [inline packages/ui/src/components/JoinForm/JoinForm.tsx:217]: Wrap this section with `Stack` like other platform jobs section and show only when there are foundationJobs:

```tsx
{foundationJobs.length > 0 && <Stack ...}
```

Comment (MustafaTRKYLMZ) [reply packages/ui/src/components/JoinForm/JoinForm.tsx:217]: We don't have any relation between jobs and foundation. Is it fixed here or not? one to many -maybe

Comment (7alip) [reply packages/ui/src/components/JoinForm/JoinForm.tsx:217]: @MustafaTRKYLMZ If a job has no platform relation, we assume that it is related to the foundation.

Comment (MustafaTRKYLMZ) [reply packages/ui/src/components/JoinForm/JoinForm.tsx]: Definitly, For example network coordinator (state members) don't have any platform.

Comment (MustafaTRKYLMZ) [reply packages/ui/src/components/JoinForm/JoinForm.tsx]: > Definitly, For example network coordinator (state members) don't have any platforms.

It means we have to add foundation (it must be optional) relations to the profiles.

**Review by 7alip — APPROVED** [review]
Thanks

**Comment by 7alip** [comment]
@ertugrul-aydin This can be merged

**Pull request merged**